deployment/terraform/engine.go

```diff
@@ -61,6 +63,27 @@ func (t *Terraform) runAWSCommand(ctx context.Context, args []string, dst io.Wri
 	return _runCommand(cmd, dst)
 }
 
+func buildGoPackage(ctx context.Context, pkg string, dst io.Writer) (string, error) {
```

Comment:
Can't we just download the release binary? Compiling code during terraform create doesn't sound ideal.